config/locales/sv.yml

```diff
@@ -0,0 +1,643 @@
+      talks: samtal
+      voice_chat: röstchatt
+      training: utbildning
+      directory: katalog
+      self_injury: självskada
+      treatment: behandling
+      care_planning: vårdplanering
+      depression: depression
+      ocd: OCD
+      advice: råd
+      forum: forum
+      learning: inlärning
+      crisis_prevention: krishantering
+      safety_planning: säkerhetsplanering
+      suicide_prevention: förebyggande av självmord
+      android: android
+      ios: iOS
+      support_groups: stödgrupper
+      mindfulness: mindfulness
+      journaling: journalföring
+      mood_tracking: humörspårning
+
+ profile:
+  index:
+   title: 'Profil'
+   edit_user: 'Anpassa profilen'
+
+ pusher:
+  not_authorized: 'Inte behörig'
+
+ search:
+  search: 'Sök'
+  search_by_name: 'Sök på namn'
+  form:
+   search_by_location: 'Sök efter plats'
+   search_by_email: 'Sök via e-mail'
+
+  index:
+   title: 'Sök efter Allierade'
+   user_not_found: 'Användaren %{email} hittades inte'
+
+ shared:
+  footer:
+    licence_name: 'AGPL-3.0'
+    license: 'Vi &hearts; FOSS'
+    foss: 'Fri och öppen källa programvara'
+    license_subtitle: 'Licensierad under %{license}'
+    connect: 'ansluta'
+  comments:
+    share_everyone: 'Dela med alla'
+    share_with: 'Dela med %{name} bara'
+    visible_only_between_you_and: 'Visas endast mellan dig och %{name}'
+    not_allies: '(inte allierade)'
+  header:
+   signout: 'Logga ut'
+   home: 'Hem'
+  viewers_indicator:
+   disabled_comments: 'Kommentarer har inaktiverats.'
+   you_are_viewer_html:
+    one: 'Du är den enda tittaren.'
+    other: 'Du är <strong>inte</strong> den enda tittaren.'
+   viewers_html:
+     zero: 'Ditt finns <strong>inga</strong> tittare.'
+     one: '%{names} är en tittare.'
+     other: '%{names} är tittare.'
+  stats:
+    you_are_focusing: 'Du fokuserar mest på'
+    user_is_focusing: '%{name} fokuserar mest på'
+  viewers:
+    plural: 'Tittare'
+    you: 'Bara du'
+    you_link: 'Synlig för dig bara'
+    many: 'Synlig för %{viewers}'
+  character_count: 'Du har %{number} tecken kvar.'
+  meeting_info:
+    attending: 'Du deltar. Ändra dig?'
+    not_attending: 'Du deltar inte. %{join}'
+    not_attending_one_spot_left: 'Du deltar inte. Det finns en plats kvar att fylla! %{join}'
+    not_attending_spots_left: 'Du deltar inte. Det finns %{number} ställen kvar att fylla! %{join}'
+    not_attending_no_spots_left: 'Du deltar inte. Det finns inget utrymme att gå med.'
+  quick_create:
+   new: 'Ny %{name}'
+
+ strategies:
+  singular: 'Strategi'
+  plural: 'Strategier'
+  edit_strategy: 'Redigera Strategi'
+  tagged_strategies: 'Taggade Strategier'
+  new: 'Ny Strategi'
+  form:
+   describe: 'Beskriv strategin.'
+   strategies_label: 'Vilka strategier skulle hjälpa till att uppnå dessa tankar?'
+   name_hint: 'Eventuella aktiviteter, tankegång eller form eller självomsorg som hjälper dig'
+   viewers_hint: 'Allierade som kan se din strategi och tagga egna ögon med det'
```

Comment:
This one looks a little fishy.

Reply:
haha yeah it does. It's meant to be "Allies who can view your strategy and tag their own moments with it"

I wanted to use the word "ögonblick" but I guess that literally means eye blink.

I've changed ögon to ögonblicken? Is that correct? Sounds better than taking your own eyes. 😅